package.json

```diff
@@ -4,6 +4,7 @@
   "version": "3.0.1",
   "description": "Sass design-systems management and utilities.",
   "homepage": "https://www.oddbird.net/accoutrement/",
+  "type": "module",
```

Comment:
For development, this will now require node v12 or higher.

Reply:
@jgerigmeyer should we add an nvmrc to help ensure that?